npm_and_yarn/lib/dependabot/npm_and_yarn/update_checker/registry_finder.rb

```diff
@@ -208,34 +215,40 @@ def unique_registries(registries)
           end
         end
 
-        # rubocop:disable Metrics/PerceivedComplexity
         def global_registry
           return @global_registry if defined? @global_registry
 
+          @global_registry ||= configured_global_registry || "https://registry.npmjs.org"
+        end
+
+        # rubocop:disable Metrics/PerceivedComplexity
+        def configured_global_registry
+          return @configured_global_registry if defined? @configured_global_registry
+
           npmrc_file&.content.to_s.scan(NPM_GLOBAL_REGISTRY_REGEX) do
             next if Regexp.last_match[:registry].include?("${")
 
-            return @global_registry = Regexp.last_match[:registry].strip
+            return @configured_global_registry = Regexp.last_match[:registry].strip
```

Comment:
Would it make sense to extract `Regexp.last_match[:registry].strip` to a method?